lib/screens/auth/forgot_pwd/forgot_pwd.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+import 'package:mobileapp/screens/auth/forgot_pwd/body.dart';
+
+class ForgotPasswordScreen extends StatelessWidget {
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Forgot Password"),
+      ),
+      body: ForgotPwdBody(),
```

Comment:
@kobbydiscount  Naming this component ForgotPasswordBody() will be ideal, Instead of ForgotPwdBody().